addons/refuel/XEH_postInit.sqf

```diff
@@ -9,3 +9,16 @@ if (isServer) then {
 [QGVAR(resetLocal), {
     _this call FUNC(resetLocal);
 }] call CBA_fnc_addEventHandler;
+
+
+#ifdef DEBUG_MODE_FULL
+diag_log text format ["[ACE-refuel] Showing CfgVehicles with vanilla transportFuel"];
+private _fuelTrucks = configProperties [configFile >> "CfgVehicles", "(isClass _x) && {(getNumber (_x >> 'transportFuel')) > 0}", true];
```

Comment:
Just a FYI im dropping here, because I made the suggestion to use `configProperties` the last time.
You could also use `"getNumber (_x >> 'transportFuel') > 0" configClasses (configFile >> "CfgVehicles")`
here and it would find everything while being faster.
This because `CfgVehicles` itself has no parents. Therefore it's pointless to use the 3rd parameter flag. All it would do is to additionally search through the parents of `CfgVehicles`. There are none though.